src/verification/verification.ts

```diff
@@ -246,11 +325,21 @@ export class UbirchVerification {
       return [];
     }
 
-    const ubirchBlxTxAnchor: IUbirchBlockchainAnchor[] = blxTXs
+    const ubirchBlxTxAnchors: IUbirchBlockchainAnchor[] = blxTXs
       .map((rawAnchor: IUbirchBlockchainAnchorRAW) => this.createBlockchainAnchor(rawAnchor))
       .filter((probablyAnchor: IUbirchBlockchainAnchor) => probablyAnchor !== undefined);
 
-    return ubirchBlxTxAnchor;
+    return ubirchBlxTxAnchors;
+  }
+
+  protected findFirstAnchorTimestamp(ubirchBlxTxAnchors: IUbirchBlockchainAnchor[]): string | null {
+    const timestamps = ubirchBlxTxAnchors
+      .map(({ timestamp }) => timestamp)
```

Comment:
maybe `.map(({ timestamp }) => Date.parse(timestamp))` to avoid unnecessary loops?

Reply:
i prefer atomizing layers of abstraction, but ok.